package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "news-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config webpack.prod.js",
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open --config webpack.dev.js"
+  },
+  "author": "ivashkovaliza",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.6.4",
```

Comment:
automatic update of node modules can unexpectedly break your app
you can use the exact version for npm packages instead of ^ operator

Reply:
done